tools/build_tar/buildtar.go

```diff
@@ -559,6 +567,33 @@ func (f *fileMeta) getMode(fname string) os.FileMode {
 	return f.defaultMode
 }
 
+type timeString time.Time
+
+var y2k = time.Date(2000, 1, 1, 0, 0, 0, 0, time.UTC)
+
+const portableTime = "portable"
+
+func (ts *timeString) String() string {
+	t := time.Time(*ts)
+	if t.Equal(y2k) {
+		return portableTime
+	}
+	return strconv.FormatInt(t.Unix(), 10)
+}
+
+func (ts *timeString) Set(v string) error {
+	if v == portableTime {
+		*ts = timeString(y2k)
+		return nil
+	}
+	unix, err := strconv.ParseInt(v, 10, 64)
+	if err != nil {
+		return fmt.Errorf("not an integer or %s: %v", portableTime, v)
```

Comment:
what if `v` is empty string (e.g. older bazel)?

Reply:
Done